config/logstash.yml

```diff
@@ -286,8 +286,17 @@
 # log.level: info
 # path.logs:
 #
-
-
+# ------------ Password Policy --------------
+# password_policy.mode: WARN or ERROR
```

Comment:
Given that this only applies for basic authentication on the API HTTP endpoint, we should change this setting for context, such as:

api.auth.basic.password_policy: WAR or ERROR